src/components/connect.js

```diff
@@ -279,6 +279,7 @@ function connect(mapPropsToRequestsToProps, defaults, options) {
         } else {
           const request = mapping.buildRequest(mapping)
           meta.request = request
+          meta.component = this.refs.wrappedInstance
```

Comment:
this should probably be done right after line 267 (or something like `const meta = Object.assign(meta, {component: this.refs.wrappedInstance})` so that identity requests and regular fetches (i.e. the branches of the `if/else`) are treated the same in this regard.